sim/warrior/deep_wounds.go

```diff
@@ -20,7 +20,7 @@ func (warrior *Warrior) applyDeepWounds() {
 		ProcMask:    core.ProcMaskMeleeMHSpecial,
 		Flags:       core.SpellFlagNoOnCastComplete,
 
-		DamageMultiplier: 1 * (1 + 0.16*float64(warrior.Talents.DeepWounds)),
+		DamageMultiplier: 1 * 0.16 * float64(warrior.Talents.DeepWounds),
```

Comment:
Just set the DamageMultiplier to 1, and I guess the ProcMask should be core.ProcMaskEmpty (?)